src/main/scala/workflow/ConcretePipeline.scala

```diff
@@ -119,9 +129,9 @@ private[workflow] class ConcretePipeline[A, B](
     }
   }
 
-  override def apply(in: A): B = apply(in, Some(Optimizer))
+  override def apply(in: A): B = apply(in, Some(DefaultOptimizer))
 
-  override def apply(in: RDD[A]): RDD[B] = apply(in, Some(Optimizer))
+  override def apply(in: RDD[A]): RDD[B] = apply(in, Some(DefaultOptimizer))
 
   def apply(in: A, optimizer: Option[RuleExecutor]): B = {
```

Comment:
Do we still want this to be an `Option[RuleExecutor]`now that we have an `Optimizer` type?